app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/SuggestionsWidget.java

```diff
@@ -113,8 +113,8 @@ protected void initializeWidgetPlacement(WidgetPlacement aPlacement) {
     }
 
     @Override
-    public void show() {
-        super.show(false);
+    public void show(@ShowFlags int aShowFlags) {
+        super.show(CLEAR_FOCUS);
```

Comment:
Shouldn't this be `super.show(aShowFlags);`?

Reply:
Should be ok. show(false) focus parameter should be equivalent to show(CLEAR_FOCUS)